ui/src/components/table/groupings-table-skeleton.tsx

```diff
@@ -21,9 +21,9 @@ const GroupingsTableSkeleton = () => {
                     <TableRow>
                         {GroupingsTableColumns.map((column) => (
                             <TableHead
-                                key={column.accessorKey}
+                                key={column.id}
                                 className={`pl-[0.5rem]
-                                ${column.accessorKey !== 'name' ? 'hidden sm:table-cell' : ''}`}
+                                ${column.id !== 'name' ? 'hidden sm:table-cell' : ''}`}
```

Comment:
One last thing, replace `column.id !== 'name'` with `column.getIndex() > 0` so it is consistent with `groupings-table.tsx`. Same change for line 40, thanks.